conformance/lang/values-types-and-variables/nil-literal.balt

```diff
@@ -0,0 +1,173 @@
+Test-Case: output
```

Comment:
Should be part of the `expressions` not `values-types-and-variables`

Reply:
will correct it.